pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -148,11 +148,14 @@ func newAlertmanager(ctx context.Context, orgID int64, cfg *setting.Cfg, store A
 	return am, nil
 }
 
-func (am *alertmanager) Ready() bool {
+func (am *alertmanager) Ready(_ context.Context) error {
```

Comment:
We don't need a context for the local Alertmanager, but we still need to implement the `Alertmanager` interface